internal/app/api/api.go

```diff
@@ -404,6 +335,56 @@ func (a *Client) DrainFeed(ctx context.Context, request *GetFeedRequest, feedFn
 	return nil
 }
 
+// GetDeletedInspections returns response from AccountsActivityLog or error
+func (a *Client) GetDeletedInspections(ctx context.Context, request *GetAccountsActivityLogRequest) (*GetAccountsActivityLogResponse, error) {
+	sl := a.sling.New().
+		Post(request.URL).
```

Comment:
Any reason we are pulling the URL in from the request parameter if it is static?

Reply:
Nope, I will make a const